Sources/Widgets/Widgets3D/SplineWidget/behavior.js

```diff
@@ -23,6 +23,30 @@ export default function widgetBehavior(publicAPI, model) {
 
   // --------------------------------------------------------------------------
 
+  publicAPI.setPixelScale = (pixelScale) => {
+    model.pixelScale = pixelScale;
+  };
+
+  // --------------------------------------------------------------------------
+
+  publicAPI.updateHandlesSize = () => {
+    if (model.scaleHandlesInPixelSpace) {
+      const scale =
+        model.pixelScale *
+        vec3.distance(
+          model.openGLRenderWindow.displayToWorld(0, 0, 0, model.renderer),
+          model.openGLRenderWindow.displayToWorld(1, 0, 0, model.renderer)
```

Comment:
why not take the moveHandle true display position ?

Reply:
How about: 
```
vec3.distance(
          model.openGLRenderWindow.displayToWorld(moveHandle.getDisplayPosition() - [handleSize/2, handleSize/2], model.renderer),
          model.openGLRenderWindow.displayToWorld(moveHandle.getDisplayPosition() + [handleSize/2, handleSize/2], model.renderer)
```

Reply:
This method does not exists in vtk-js.